table/block_based/filter_block_reader_common.cc

```diff
@@ -79,6 +79,7 @@ Status FilterBlockReaderCommon<TBlocklike>::GetOrReadFilterBlock(
   if (no_io) {
     read_options.read_tier = kBlockCacheTier;
   }
+  read_options.verify_checksums = false;
```

Comment:
Oh, this is the same as https://github.com/facebook/rocksdb/pull/10438/files#r932893036. I really hope someone can plumb the `ReadOptions` to this code path. cc @gitbw95

Reply:
It seems I was able to manage this one out.